source/geometry_model/chunk.cc

```diff
@@ -728,6 +728,10 @@ namespace aspect
 
               AssertThrow (point1[2] < point2[2],
                            ExcMessage ("Minimum latitude must be less than maximum latitude."));
+              AssertThrow (point1[2] > -0.5*numbers::PI,
+                           ExcMessage ("Minimum latitude cannot be exactly -90 degrees"));
```

Comment:
maybe "... needs to be larger than -90 degrees" would be a better error message? Can you add a period at the end?